From d3af39c1afafa11d71ae374a7031e048f5829e3c Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 16 Feb 2008 11:46:30 +0200
Subject: [PATCH] zoo.c: (ms_help): Reduce outputted newlines in help screens

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 zoo.c |   11 +++++------
 1 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/zoo.c b/zoo.c
index 26de52a..2191a43 100644
--- a/zoo.c
+++ b/zoo.c
@@ -285,7 +285,6 @@ give_list:
 
 /* help screen */
 bigusage:
-printf("\n\n\n\n\n\n\n\n");
 printf ("Zoo archiver, %s\n", version);
 printf("(C) Copyright 1991 Rahul Dhesi -- Noncommercial use permitted\n");
 
@@ -293,13 +292,13 @@ printf (usage);
 printf ("\nChoose a command from within {} and zero or more modifiers from within [].\n");
 
 printf ("E.g.:  `zoo a save /bin/*' will archive all files in /bin into save.zoo.\n");
-printf ("(Please see the user manual for a complete description of commands.)\n\n");
+printf ("(Please see the user manual for a complete description of commands.)\n");
 printf (nov_usage);
 printf (nov_cmds);
-printf ("\n\n\n\n");
+printf ("\n");
 wait_return();	/* print msg & wait for RETURN */
 
-printf ("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n");
+printf ("\n");
 printf (usage);
 
 printf (" Commands in {} mean:         |Modifiers in [] mean:\n");
@@ -484,7 +483,7 @@ char *options;
 
 	/* loop thru screens */
 	for (curscreen = screens; *curscreen != NULL; ++curscreen) {
-		printf("\n\n\n\n");
+/* 		printf("\n\n\n\n"); */
 		linecount = scrnlen;
 		curline = *curscreen;
 		while (*curline != NULL) {
@@ -493,7 +492,7 @@ char *options;
 		}
 
 		/* slew page */
-		while (--linecount != 0) putchar('\n');
+/* 		while (--linecount != 0) putchar('\n'); */
 		wait_return();	/* print msg & wait for RETURN */
 	}
 
-- 
1.5.3.8

