From 0f1ded27f135f8302e4ea123175dae1f11154af1 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 16 Feb 2008 11:01:45 +0200
Subject: [PATCH] zoo.c: (wait_return): comment out

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 zoo.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/zoo.c b/zoo.c
index 26de52a..fd9d8d7 100644
--- a/zoo.c
+++ b/zoo.c
@@ -511,6 +511,7 @@ static void wait_return()
 	if (!isatty(fileno(stdout)) || !isatty(fileno(stdin)))
 		return;
 #endif
+#ifdef PAUSE_HELP
 	(void) printf("Hit RETURN (or ENTER) key to continue...");
 	for ( ; ;) {
 		int key; 
@@ -520,4 +521,5 @@ static void wait_return()
 		if (key == '\n' || key == '\r')
 			return;
 	}
+#endif
 }
-- 
1.5.3.8

